application.example.yaml

```diff
@@ -6,6 +6,7 @@
 # Matrix config items #
 #######################
 # Matrix domain, same as the domain configure in your Homeserver configuration.
+# (note: Homeserver domain may be defined as 'server_name' in configuration file).
```

Comment:
We talk about Matrix domain on the line above, and now we talk about Homeserver domain. this can be missleading.
We should stick with "Matrix domain...".

`server_name` is specific to synapse, we should ensure it is explicitly added.

Reply:
I agree with you, and a recent discussion in ```#mxisd:kamax.io``` room showed that people may be confused regarding the term "*domain*" - while Synapse mentions "*server name*" which typicallly leads people to type the hostname of the server.

Is it ok if I change the comments by:
*# (note: **in Synapse Homeserver**, the Matrix domain may be defined as 'server_name' in configuration file).* 
?

Reply:
sounds good, yes